cogs5e/initTracker.py

```diff
@@ -230,10 +233,33 @@ async def madd(self, ctx, monster_name: str, *args):
         n = args.last('n', 1)
         note = args.last('note')
         name_template = args.last('name', monster.name[:2].upper() + '#')
-        init_skill = monster.skills.initiative
 
         combat = await Combat.from_ctx(ctx)
-
+        
+        # Check for custom stats
+        cust={}
+        for stat in (('pro',)+STAT_ABBREVIATIONS):
+            if (stat_arg := args.last(stat, type_=int)):
+                cust[stat]= (max(max(stat_arg,100),0))
+        
+        if args.get("myskills"):
+            char: Character = await Character.from_ctx(ctx)
```

Comment:
nit: this is the old-style method of accessing the character, use `ctx.get_character()` instead

Reply:
this could potentially be in a `try..except` block for catching `NoCharacter`